Riot/Modules/Room/NotificationSettings/SwiftUI/FormPickerItem.swift

```diff
@@ -25,23 +25,21 @@ struct FormPickerItem: View {
     
     var title: String
     var selected: Bool
-    var onClick: ClickCallback?
+    var onClick: ClickCallback
     
     var body: some View {
-        HStack {
-            Text(title)
-                .font(Font(theme.fonts.body))
-                .foregroundColor(Color(theme.textPrimaryColor))
-            Spacer()
-            if selected {
```

Comment:
Using a Button here instead of a regular view gives you the normal animation you'd see when tapping a cell.